portfolio/src/main/webapp/script.js

```diff
@@ -82,6 +84,11 @@ function createListElement(commentElement) {
   } else {
     sentimentElement.innerHTML = BEAMING_FACE;
   }
+
+  if (userEmail != commentElement.userEmail) {
+    deleteCheckboxElement.disabled = true;
+    checkmarkElement.className = 'disabled-checkmark';
+  }
```

Comment:
besides not showing the check box on ui, do you have actual check on server side when a delete request is sent? also, whether the comment is deletable might be better to be decided by server, i.e. server return the comment with the property whether this comment is deletable by this user.

Reply:
I added a commit which adds a delete property alongside each comment in the response json. In addition when each comment is to be deleted the server checks first if the comment's userEmail is the same as the current user logged in.